site/src/index.css

```
@@ -19,23 +19,32 @@ code {
 }
 
 :root {
-  --peterportal-primary-color-1: #2484c6;
-  --peterportal-primary-color-2: #74d1f6;
-  --peterportal-secondary-green: #81c284;
-  --peterportal-secondary-yellow: #f5d77f;
-  --peterportal-secondary-orange: #ecad6d;
-  --peterportal-secondary-red: #e7966d;
-  --peterportal-light-gray: #eeeeee;
-  --peterportal-mid-gray: #8d8d8d;
-  --peterportal-uci-gold: #ffd200;
-  --peterportal-gray-blue: #f5f6fc;
-  --zot-blue: var(--peterportal-primary-color-1);
-  --aldrich-blue: var(--peterportal-primary-color-2);
+  --blue-primary: #2484c6;
+  --blue-secondary: #5babe1;
+  --blue-tertiary: #a0ceee;
+  --blue-accent: #74d1f6;
+
+  --green-primary: green;
+  --green-secondary: #81c284;
+  --yellow-secondary: #f5d77f;
+  --orange-secondary: #ecad6d;
+  --red-secondary: #e7966d;
+
+  --mid-gray: #8d8d8d;
+  --gray-blue: #f5f6fc;
   --ring-road-white: #ffffff;
-  --petr-gray: #808080;
+
   --border-radius: 8px;
 }
 
+@media (prefers-color-scheme: dark) {
+  :root {
+    --blue-secondary: #185580;
+    --blue-tertiary: #0b283c;
+    --green-secondary: #3e7f41;
+  }
+}
+
```

In response to the other comment about matching peterportal theme instead of system theme
```suggestion
body[data-theme="dark"] {
  --blue-secondary: #185580;
  --blue-tertiary: #0b283c;
  --green-secondary: #3e7f41;
}
```

Note that this would override it on the body rather than the `:root`, but that should not be a big deal in this case. However, it is something that should be kept in mind.

---

Sounds good. Thoughts on leaving a comment within `index.css`, saying `/* Keep in mind that these colors are defined in :root but overridden in body */`, right above like 40? Or should we just remember ourselves and leave it out to keep the code cleaner